Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Person.hs

```diff
@@ -94,12 +94,13 @@ data PersonE e = Person
     deviceToken :: Maybe FCM.FCMRecipientToken,
     description :: Maybe Text,
     merchantId :: Id DMerchant.Merchant,
+    isSimulated :: Bool,
     whatsappNotificationEnrollStatus :: Maybe Whatsapp.OptApiMethods,
     referralCode :: Maybe Text,
     referredAt :: Maybe UTCTime,
     hasTakenValidRide :: Bool,
-    blockedAt :: Maybe UTCTime,
-    blockedByRuleId :: Maybe (Id DMC.MerchantConfig),
+    actionTakenAt :: Maybe UTCTime,
+    actionRuleId :: Maybe (Id DMC.MerchantConfig),
```

Comment:
Maybe it would be better to have some entry in another table instead of having these maybes?